nestkernel/eprop_archiving_node_readout.h

```diff
@@ -0,0 +1,145 @@
+/*
+ *  eprop_archiving_node_readout.h
```

Comment:
Just for my understanding: Why are some of these functions not included in an `eprop_archiving_node_readout_impl.h` file, analogous to `eprop_archiving_node_recurrent_impl.h`?